toolchains/install_bootstrap/musl/setup.sh

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env bash
```

Comment:
SGTM, maybe we should change the name to be a little more descriptive? Something like `mktoolchain.sh`?